code/game/gamemodes/gang/gang.dm

```diff
@@ -248,6 +249,19 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
+	if(forced_shuttle)
+		return
+	var/alive = 0
+	for(var/mob/living/L in GLOB.player_list)
+		if(L.stat != DEAD)
+			alive++
+	if(alive < LAZYLEN(GLOB.joined_player_list) * 0.4)
+		priority_announce("Catastrophic casualties detected: crisis shuttle protocols activated - jamming recall signals across all frequencies.")
```

Comment:
The only conditional checks before this are for the crew alive and time since last `/gang_points/process()` call. This announcement could occur as the shuttle docks with the station. Or when it's at centcom, the round has ended and there's 2 seconds left before the server restarts; which is quite possible given the usual post-round deathmatch.